wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_generic_10.xml

```diff
@@ -29,15 +29,15 @@
 				<argument name="system_name" value="search_element"/>
 			</Arguments>
 		</step>
-		<step TS= '3' Driver='selenium_driver' Keyword='verify_text' >
-			<Arguments>
-				<argument name="system_name" value="search_element"/>
-                <argument name="locator" value="id-search-field"/>
-				<argument name="locator_type" value="id"/>
-				<argument name="var" value="searchtext"/>
-				<argument name="expected" value="Search"/>
-			</Arguments>
-		</step>
+		<!--<step TS= '3' Driver='selenium_driver' Keyword='verify_text' >-->
```

Comment:
Hi @apathapa-warrior,

Is there a reason why this step was commented out?

Reply:
Looks like the corresponding py2 PR was merged. I will create a ticket to track this commented out step.

Reply:
Created WAR-2111 to track this. Resolving comments.